installer/conf/azm-containers-parser.conf

```diff
@@ -0,0 +1,18 @@
+[PARSER]
```

Comment:
Could you remove the empty file, adding comment here since it doesnt let me comment on that.

Reply:
I assume, you are referring azm-containers-parser.conf fie.  If Yes, this is not empty file.

Reply:
No, i am referring to the file above this in this PR. The name of the file is ,

Reply:
https://github.com/microsoft/Docker-Provider/blob/545a6ff72605be48e42535126c94df1f527b98f4/%2C

Reply:
> https://github.com/microsoft/Docker-Provider/blob/545a6ff72605be48e42535126c94df1f527b98f4/%2C

> thanks for catching empty file. just deleted it. :)